### PR TITLE
Fixes staff of lava working on space tiles

### PR DESCRIPTION
**Pull request by Time-Green**
This makes no sense

:cl:
fix: Staff of laval no longer works on space
/:cl:

### CODE REVIEW

File: code/modules/mining/lavaland/megafauna_loot.dm [hunk 1]
```diff
@@ -777,7 +777,7 @@
 	var/create_delay = 3 SECONDS
 	var/reset_cooldown = 5 SECONDS
 	var/timer = 0
-	var/static/list/banned_turfs = typecacheof(list(/turf/open/space/transit, /turf/closed))
+	var/static/list/banned_turfs = typecacheof(list(/turf/open/space/transit, /turf/closed, /turf/open/space))
```
[the remark this answers is not in the document]
Comment (Time-Green) [reply code/modules/mining/lavaland/megafauna_loot.dm:780]: if you prefer, sure

**Comment by the-orange-cow** [comment]
Alternative idea: lava staff in space makes an asteroid block

**Comment by Elli-Skala** [comment]
F in chats to being able to make the sm 27k cooling loop into a 5000k heating loop

**Comment by YakumoChen** [comment]
Coder God (Oranges): What are the spacemen doing out there?
Chaplain: They are making lava from the void
God: What? I gave them a whole lava z level to make lava on.
Chaplain: They don't like that level.
God [Mockingly]: ThEy DoNt LiKe ThAT LeVeL *forcecommits a nerf*

**Comment by Whoneedspacee** [comment]
Counterpoint, magic

**Pull request merged**